scripts/mbedtls_framework/config_checks_generator.py

```diff
@@ -218,9 +218,9 @@ def main(branch_data: BranchData) -> None:
                         help='output file location (default: %(default)s)')
     options = parser.parse_args()
     list_only = options.list or options.list_for_cmake
-    output_files = generate_header_files(branch_data,
-                                         options.output_directory,
-                                         list_only=list_only)
+    output_files = list(generate_header_files(branch_data,
```

Comment:
The fact that we all missed this at first shows that this is not a good way to write the code. But right now I'd rather unblock the chain of PR ASAP.